docs/src/pages/landing/Users.js

```diff
@@ -82,68 +81,36 @@ const users = [
   },
 ];
 
-const useStyles = makeStyles(
-  (theme) => ({
-    root: {
-      padding: theme.spacing(2),
-      minHeight: 160,
-      paddingTop: theme.spacing(5),
-    },
-    container: {
-      marginBottom: theme.spacing(4),
-    },
-    users: {
-      padding: theme.spacing(10, 4, 0),
-    },
-    grid: {
-      marginTop: theme.spacing(5),
-      marginBottom: theme.spacing(5),
-    },
-    img: {
-      margin: theme.spacing(1.5, 3),
-    },
-    amazon: {
-      margin: theme.spacing(2.4, 3, 1.5),
-    },
-    coursera: {
-      margin: theme.spacing(2.3, 3, 1.5),
-    },
-    unity: {
-      margin: theme.spacing(0.5, 3, 1.5),
-    },
-    walmart: {
-      margin: '13px 4px 12px',
-    },
-    button: {
-      margin: theme.spacing(2, 0, 0),
-    },
-  }),
-  { name: 'Users' },
-);
+const styles = {
+  amazon: { mt: 2.4 },
+  coursera: { mt: 2.3 },
+  unity: { mt: 0.5 },
+  walmart: { mt: '13px', mx: '4px', mb: '12px' },
+};
 
 export default function Users() {
-  const classes = useStyles();
   const t = useTranslate();
 
   return (
-    <div className={classes.root}>
+    <Box sx={{ p: 2, minHeight: 160, pt: 5 }}>
       <NoSsr defer>
-        <Container maxWidth="md" className={classes.container} disableGutters>
+        <Container sx={{ mb: 4 }} maxWidth="md" disableGutters>
           <Divider />
-          <div className={classes.users}>
+          <Box sx={{ pt: 10, px: 4 }}>
             <Typography variant="h4" component="h2" align="center" gutterBottom>
               {t('whosUsing')}
             </Typography>
             <Typography variant="body1" align="center" gutterBottom>
               {t('joinThese')}
             </Typography>
-            <Grid container justifyContent="center" className={classes.grid}>
+            <Grid container justifyContent="center" sx={{ my: 5 }}>
               {users.map((user) => (
-                <img
+                <Box
+                  component="img"
                   key={user.caption}
                   src={`/static/images/users/${user.logo}`}
                   alt={user.caption}
-                  className={clsx(classes.img, classes[user.class])}
```

Comment:
Hi @oliviertassinari @vicasas 

in this case can we add this library @emotion/css and 
**Sample code**
```

import { cx, css } from '@emotion/css'

const cls1 = css`
  font-size: 20px;
  background: green;
`
const cls2 = css`
  font-size: 20px;
  background: blue;
`

<div className={cx(cls1, cls2)} />
```
@oliviertassinari 
emotion provide number library which can useful for styling or in latest material ui provide will those option?